src/Calculator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Acme;
+
+use DateTimeImmutable;
+
+class Calculator
+{
+    public $displayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaaayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy;
+
+    public function div(int $a, int $b): int
```

Comment:
**[phpmd]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Avoid variables with short names like $a. Configured minimum length is 3.